lib/sequencescape-api/connection_factory.rb

```diff
@@ -7,8 +7,8 @@ class Sequencescape::Api::ConnectionFactory
 
   def self.create(options)
```

Comment:
Where is this method called from?

Reply:
https://github.com/JamesGlover/sequencescape-client-api/blob/087bbd8d29359c56272e2ba9bcd36c72f06aa048/lib/sequencescape-api/core.rb#L16

Reply:
Ugh, last time I tried that it did a nice inline preview.

Reply:
Ahh, needs to be the same repo

Reply:
Ah thanks, I was searching for `Sequencescape::Api::ConnectionFactory.create`